core/base/document.php

```diff
@@ -1151,14 +1151,16 @@ public function delete() {
 	}
 
 	/**
-	 *
 	 * @since 3.6.0
 	 *
 	 * @param array $config
-	 *
 	 * @param array $map_old_new_post_ids
+	 *
+	 * @deprecated 3.8.0 Use `::on_import_update_ids()` instead.
 	 */
 	public static function on_import_replace_dynamic_content( $config, $map_old_new_post_ids ) {
+		Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_function( __METHOD__, '3.8.0', __CLASS__ . '::on_import_update_ids()' );
```

Comment:
Same, and in any other place.
```suggestion
		Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_function( __METHOD__, '3.8.0', __CLASS__ . '::on_import_update_dynamic_content()' );
```

Reply:
Done.